spec/factories.rb

```diff
@@ -12,4 +12,19 @@
     application_form
     status { :application_complete }
   end
+
+  factory :personal_details_form, class: CandidateInterface::PersonalDetailsForm do
```

Comment:
If we're only using this in a single place, it might be better to move this into `personal_details_form_spec.rb`. I'm also wondering if it's clearer to use `CandidateInterface::PersonalDetailsForm.new` instead of a factory.

Reply:
We'll move it inside the spec 👍